files/bash-completion

```diff
@@ -31,7 +31,12 @@ __fail2ban_jail_action_methods () {
 
 _fail2ban () {
     local cur prev words cword
-    _init_completion || return 
+    _init_completion || return
+
+    if [[ "$1" == *"fail2ban-regex" ]] && [[ "$cur" == "-"* ]];then
```

Comment:
tbh i don't know anymore (too long ago).
Surely id made sense back then => just close the PR ;-)